### PR TITLE
[14.0][MIG] fieldservice_geoengine

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

**Comment by AmmarOfficewalaSerpentCS** [comment]
https://github.com/OCA/geospatial/pull/292

Dependent on base_geoengine module

**Comment by max3903** [comment]
@Freni-OSI Can you fix the Travis errors?

**Comment by Freni-OSI** [comment]
> @Freni-OSI Can you fix the Travis errors?

the dependency PR is closed that's why it is failing. need to migrate base_geoengine again.
CC: @max3903

**Comment by Freni-OSI** [comment]
@max3903 , Could you please take a look at the test failures?

**Comment by nikul-serpentcs** [comment]
Hello @max3903 @brian10048 

Here **Mr.Travis** is failed because of packages, How can we add packages like this? 

https://github.com/OCA/field-service/pull/654/commits/21e76280ab487467807008a1e41d2fa8310c89f4#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R15-R16

**Comment by Urvisha-OSI** [comment]
@max3903 here tests are fails because of fieldservice_account_analytic module test cases, do we need to change that in this PR?

**Comment by max3903** [comment]
@Urvisha-OSI Yes

**Review by max3903 — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
             name: test with OCB
     services:
       postgres:
-        image: postgres:9.6
+        image: ursa/postgresql:latest
```
Comment (HekkiMelody) [inline .github/workflows/test.yml]: Is this an appropriate change?

Also, shouldn't we be using postgis, which has an official image over on docker hub
There is no maintained 9.6 image, but the oldest still maintaned image is [postgis/postgis:10-3.2](https://registry.hub.docker.com/r/postgis/postgis/tags?page=1&name=10-3.2)

Comment (max3903) [reply .github/workflows/test.yml]: @aleuffre Yes it is appropriate, even if your solution is better.

**Comment by max3903** [comment]
@Urvisha-OSI Can you please fix the pre-commit errors?

**Comment by Urvisha-OSI** [comment]
> @Urvisha-OSI Can you please fix the pre-commit errors?

@max3903 all checked have passed in this PR

**Pull request merged**